src/GridFieldEditableColumns.php

```diff
@@ -250,7 +251,7 @@ public function getFields(GridField $grid, DataObjectInterface $record)
                 }
             }
 
-            if (!$field && $list instanceof ManyManyList) {
+            if (!$field && ($list instanceof ManyManyList) || ($list instanceof ManyManyThroughList)) {
```

Comment:
```suggestion
            if (!$field && ($list instanceof ManyManyList || $list instanceof ManyManyThroughList)) {
```

I think this change currently runs the risk of passing this check even when `$field` has already been set:
```php
if (false && (true) || (false)) {
```

Reply:
Good point. Done